include/ensmallen_bits/cmaes/cmaes.hpp

```diff
@@ -70,6 +71,7 @@ class CMAES
    *     objective.
    */
   CMAES(const size_t lambda = 0,
+        const double initialSigma = 0.6,
```

Comment:
I'm concerned that this would break reverse compatibility and we'd have to bump the major version (I don't think that's justified by a relatively simple change like this).  According to our versioning policy,

```
 * A MAJOR version bump implies a backwards-incompatible change to the
   user-facing optimizers.  Code using one major version of ensmallen may
   not work with a newer major version of ensmallen.
```

We could instead add `initialSigma` as the last parameter with a default value, and then we could optionally mark that constructor deprecated ("`will be removed in 2.10.0`") and add another constructor just like this one here.  What do you think?

Reply:
Yup, didn't think of that. I'll change it :)